src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -586,6 +587,93 @@ const formConfig = {
           itemFilter: item => _.get('view:selected', item),
           arrayPath: 'disabilities',
           depends: (formData, index) =>
+            !isProd &&
+            _.get(
+              `disabilities.${index}.view:selectableEvidenceTypes.view:privateMedicalRecords`,
+              formData,
+            ),
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:title': disabilityNameTitle,
+                'ui:description': privateRecordsChoice,
+                'view:uploadPrivateRecords': {
+                  'ui:title':
+                    'Do you want to upload your private medical records?',
+                  'ui:widget': 'radio',
+                  'ui:options': {
+                    labels: {
+                      yes: 'Yes',
+                      no: 'No, please get them from my doctor',
+                    },
+                  },
+                },
+                'view:patientAcknowledgement': {
+                  'ui:title': ' ',
+                  'ui:help': patientAcknowledgmentText,
+                  'ui:options': {
+                    expandUnder: 'view:uploadPrivateRecords',
+                    expandUnderCondition: 'no',
+                    showFieldLabel: true,
```

Comment:
Ahh, this is tricky. After fiddling with it for a while, I think I understand why you did this.

Correct me if I'm wrong:
- `showFieldLabel` is needed to display the validation message and `ui:help`
- `'ui:title': ' '` is needed to get rid of the gross `view:patientAcknowledgement` title
- `ui:help` is needed to put the expanding text below the check box
  - This can also be done by using another empty object after `view:patientAcknowledgement`, but that doesn't solve the validation message problem

Does that sound about right?

Reply:
That is correct.

Reply:
Perfect, thanks. I'll file an issue for it in [USFS](https://github.com/usds/us-forms-system) today.